modules/Other/PublicationEnum.yaml

```diff
@@ -0,0 +1,9 @@
+enums:
+  StudyTypeEnum:
```

Comment:
Enum name needs to match what's defined in `range` (PortalPublication.yaml line 47). Let's call this `PublicationTypeEnum` and update `range` there accordingly.